libs/domain/picking/src/components/filter-button/filter-button.def.ts

```diff
@@ -4,4 +4,6 @@ export const filterButtonComponent = componentDef({
   name: 'oryx-picking-filter-button',
   impl: () =>
     import('./filter-button.component').then((m) => m.FilterButtonComponent),
+  schema: () =>
+    import('./filter-button.schema').then((m) => m.filterButtonComponentSchema),
```

Comment:
This is true. According the designs it will be sort + filters modal in the future: https://www.figma.com/file/OgvBbM6xRMVfy6VYouxONy/PA-Spryker?type=design&node-id=1598-27429&t=cLDDDXFRvIQeG6wb-0